src/app/components/event-page/event-page.component.html

```diff
@@ -1,84 +1,96 @@
-<div class="event-card"> 
-    <div class="top-div">
-        <img class="event-image" [src]="event.backgroundImage ? event.backgroundImage : ''" alt="">
-        <div class="event-information">
-            <div class="event-information-first-row">
-                <div  class="event-text-information">
-                    <div class="title-date">
-                        <div class="event-title">{{event.title}}</div>
-                        <div class="start-date">{{ event.startingDate | date:'short' }}</div>
+<div class="event">
+    <mat-card class="event-card">
+        <div class="top-div">
+            <div class="event-image">
+                <img  [src]="event.backgroundImage != 'string' ? event.backgroundImage : 'https://media.istockphoto.com/photos/baby-husky-puppy-playing-in-the-park-picture-id989949432?k=20&m=989949432&s=170667a&w=0&h=SWbImVzYQZnWUerppitlWdqDo3rAfaztstpbqiO2Pjo='" alt="">
```

Comment:
Do we really want the puppy image to show up on every event that doesn't have a picture? :D